packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -368,6 +366,7 @@ export default class Meetings extends WebexPlugin {
     onReady() {
       this.webex.once(READY, () => {
         StaticConfig.set(this.config);
+        LoggerProxy.set(this.webex.logger);
```

Comment:
I just happened to go through [my PR and it's description](https://github.com/webex/webex-js-sdk/pull/2250) and I guess this change would affect the Node.js SDK again 🙁 

This ticket would get re-opened -> https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-263135 

Originally this line was here and in my PR, we moved the initialise even before the onReady call because, in Node.js SDK, RTCPeerConnection object won't be available and [MeetingsUtil.hasH264Codec](https://github.com/webex/webex-js-sdk/blob/master/packages/node_modules/%40webex/plugin-meetings/src/meetings/util.js#L74) would try to create one and fail. When the error is caught, Logger wouldn't be initialised already and Node.js SDK gets stuck there.

Reply:
I can see it now , The actual issue is with the client they need to wait for on ready then do register and should not simply go and do register

on the other hand if they are running on node we should not call the hasH264 function , Its better doing that way

Reply:
As discussed in the standup we need to track this separately we have so many function which are browser specific but in node we will need to disable those .

Also the developers need to wait for onready before registering